.env

```diff
@@ -72,6 +72,7 @@ OPEN_RUNTIMES_NETWORK=appwrite_runtimes
 _APP_EXECUTOR_SECRET=your-secret-key
 _APP_EXECUTOR_HOST=http://appwrite-executor/v1
 _APP_MAINTENANCE_INTERVAL=86400
+_APP_MAINTENANCE_RETENTION_CACHE=2592000
```

Comment:
We need to include this variable in variables.php and compose.phtml
https://github.com/appwrite/appwrite/blob/master/app/config/variables.php
https://github.com/appwrite/appwrite/blob/master/app/views/install/compose.phtml